app/domain/report.rb

```diff
@@ -10,7 +10,7 @@ def self.generate(*args)
   def initialize(content_items, request)
     self.content_items = content_items
     self.request = request
-    self.questions = Question.order(:id).to_a
+    self.questions = Audits::Question.order(:id).to_a
```

Comment:
Could you please move the `Report` to the `audits` module within `domain`?. After doing it, you won't need this prefix.